[READY 4 REVIEW] Fixes multiple bugs with 3/4 sprites, adds alignments to consoles as well

DeluxEdition opened the pull request:
As title.

DeluxEdition committed:
Fixes multiple bugs with 3/4 sprites, adds alignments to consoles as
well.

BlueNexus commented:
Please don't just straight-up get rid of the chunkier consoles. Helm/engine control ones are meant to look thick, since they're bulky consoles that are directly and heavily wired into the ship. Them being distinct also makes them easily recogniseable. Make a new thicc sprite for them.

DeluxEdition commented:
I'm sure we can give thicc consoles the love they deserve at some point, count me on this, but I'd like to get this merged since it fixes the grossest errors in the new sprites.

babydoll commented:
Can you give a quick run down of what was changed in each file?

DeluxEdition commented:
I misclicked, don't mind the reopen!

Sure, here goes:

- engine_control.dm, helm.dm, sensors.dm and shuttle.dm: removed the "thick" icon state for now so they use default console sprites, as @BlueNexus mentioned. This is a temporary fix until I can make a proper sprite.
- interactions.dm: changed item stacking on racks so they stack vertically instead of diagonally to fit the new sprite.
- computer.dmi and modular_console.dmi: Added proper alignment for keyboards and for computer.dmi, proper alignment for screens too. Consoles will no longer look weird when rotated due to spin view commands.
- modular_laptop.dmi: added a missing sprite: the laptop was not showing up since the proc expected "laptop-open" and the sprite was called only "laptop".
- storage.dmi: Most station crates are now in the proper 3/4 style instead of 3-point; lunchboxes as well.

I think that's about it.

comma commented:
>Please don't just straight-up get rid of the chunkier consoles

They existed because default consoles were too thin and non-masculine, now default sprite is stocky is thicc and there's little you can do (or want to) to make it thiccer. So I asked him to axe those.

Sunbeamstress commented:
*obligatory late comment about how it's 2561 and why are we arguing about the profile of consoles that realistically ought to be able to fit onto a device the thickness of a sheet of paper*

HeyBanditoz commented:
But it's 2562

On May 15, 2018 11:35:52 AM MDT, Brendon Hollesen <notifications@github.com> wrote:
>*obligatory late comment about how it's 2561 and why are we arguing
>about the profile of consoles that realistically ought to be able to
>fit onto a device the thickness of a sheet of paper*
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/Baystation12/Baystation12/pull/21484#issuecomment-389250734

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
